### PR TITLE
Remove trailing comma

### PR DESCRIPTION
**Pull request by equinox**
Trying to fix Bracket's error on install.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "brackets": ">=1.4.0"
     },
     "contributors": [
-        {"name": "Patrick Oladimeji", "email": "thehogfather@dustygem.co.uk", "url": "https://github.com/thehogfather"},
+        {"name": "Patrick Oladimeji", "email": "thehogfather@dustygem.co.uk", "url": "https://github.com/thehogfather"}
```
Comment (equinox) [inline package.json:15]: http://i.imgur.com/XLeOgrA.jpg this is the error I get on install.

**Comment by thehogfather** [comment]
Thanks

**Comment by equinox** [comment]
Have you got it working? I'm now getting http://i.imgur.com/Ba74xcA.png

**Comment by thehogfather** [comment]
yes pull the latest from master. The version number did not follow semantic versioning.  Note that this also requires the github version of brackets to work (ie Brackets >= ~~1.4.0~~ 1.3.0).

**Pull request merged**